pkg/kubelet/metrics/metrics.go

```diff
@@ -462,6 +462,26 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// GracefulShutdownStartTime is a gauge that records the time at which the kubelet is started to graceful shutdown.
+	GracefulShutdownStartTime = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           "graceful_shutdown_start_time",
+			Help:           "Graceful shutdown start time in unix timestamp in seconds. if graceful shutdown is not in progress, and this number will be 0 ",
```

Comment:
Since you are using gauges and not counters, the metrics don't have to start from 0. I would expect the metric not to exist if a graceful shutdown didn't occur.